<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgIOParameter</class>
 <widget class="QDialog" name="DlgIOParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IO参数</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_iopara">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>461</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_input">
    <attribute name="title">
     <string>输入</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_output">
    <attribute name="title">
     <string>输出</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_iobackup">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>参数备份</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_iorecover">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>参数还原</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_iosave">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ioleave">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>离开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>最大输入输出：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input_maxnum">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_output_maxnum">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
